packages/react-ai/src/components/AIConversation/views/Controls/InputControl.tsx

```diff
@@ -0,0 +1,232 @@
+    </svg>
+  ),
+  stroke: '#0D1A26',
+  ...iconAttributes,
+});
+
+const AttachIcon = withBaseElementProps(Icon, attachIconProps);
+
+const AttachFileContainer = withBaseElementProps(View, {
+  className: `${INPUT_BLOCK}__${ATTACH_FILE_BLOCK}__container`,
+});
+
+const VisuallyHidden = withBaseElementProps(View, {
+  className: `${INPUT_BLOCK}__visually-hidden`,
+});
+
+const AttachFileButton = withBaseElementProps(Button, {
+  'aria-label': 'Attach file',
+  className: `${INPUT_BLOCK}__button ${INPUT_BLOCK}__button--attach`,
+  role: 'button',
+});
+
+const AttachFileInputBase = withBaseElementProps(Input, {
+  'aria-label': 'Attach file',
+  accept: 'image/*',
+  className: `${INPUT_BLOCK}__attach-file-input`,
+  type: 'file',
+});
+
+const AttachFileInput: typeof AttachFileInputBase = React.forwardRef(
+  function AttachFileInput(props, ref) {
+    // TODO should attach file to message
+    return <AttachFileInputBase {...props} ref={ref} />;
+  }
+);
+
+const AttachFileControl: AttachFileControl = () => {
+  const hiddenInput = React.useRef<HTMLInputElement>(null);
+
+  function handleClick() {
+    if (hiddenInput.current) {
+      hiddenInput.current.click();
+      hiddenInput.current.value = '';
+    }
+  }
+
+  return (
+    <AttachFileContainer>
+      <AttachFileButton onClick={handleClick}>
+        <AttachIcon />
+      </AttachFileButton>
+      <VisuallyHidden>
+        <AttachFileInput ref={hiddenInput} />
+      </VisuallyHidden>
+    </AttachFileContainer>
+  );
+};
+
+AttachFileControl.Container = AttachFileContainer;
+AttachFileControl.Button = AttachFileButton;
+AttachFileControl.Input = AttachFileInput;
+AttachFileControl.VisuallyHidden = VisuallyHidden;
+
+interface AttachFileControl<
+  T extends Partial<AIConversationElements> = AIConversationElements,
+> {
+  (): React.JSX.Element;
+  Button: T['Button'];
+  Container: T['View'];
+  Input: T['Input'];
+  VisuallyHidden: T['View'];
+}
+
+const sendIconProps = () => ({
+  children: (
+    <svg>
+      <g clipPath="url(#clip0_387_1426)">
+        <path
+          d="M14.3199 7.98214L1.66669 1.23486L3.97674 7.98214M14.3199 7.98214L1.66669 14.5689L3.97674 7.98214M14.3199 7.98214H3.97674"
+          strokeWidth="1.5"
+          strokeLinecap="square"
+          strokeLinejoin="round"
+        />
+      </g>
+      <defs>
+        <clipPath id="clip0_387_1426">
+          <rect width="100%" height="100%" />
+        </clipPath>
+      </defs>
+    </svg>
+  ),
+  stroke: 'white',
+  ...iconAttributes,
+});
+
+const SendIcon = withBaseElementProps(Icon, sendIconProps);
+
+const SendButtonBase = withBaseElementProps(Button, {
+  'aria-label': 'Send message',
+  className: `${INPUT_BLOCK}__button ${INPUT_BLOCK}__button--send`,
+  disabled: false,
```

Comment:
A note about disabled/aria-disabled. These don't necessarily go hand-in-hand. https://css-tricks.com/making-disabled-buttons-more-inclusive/ Depending on the UX you'd probably need one or the other (probably just disabled).